src/patterns/news/index.njk

```diff
@@ -0,0 +1,114 @@
+
+### Landing page
+
+This page sorts the already-published news so that the most recently posted article is at the top of the list.
+
+Each landing page preview needs to have a:
+
+- short, clearly written title that links to the news article
+- short summary, usually taken from the first paragraph of the article, which should be no more than four lines (around 150 characters)
+- [date that follows house style](https://style.ons.gov.uk/category/writing-for-the-web/plain-english), for example, 23 September 2020
+- category, which links to the page that lists all articles in that category
+- thumbnail image
+
+You can add [related content](https://ons-design-system.netlify.app/components/related-content), which appears on the right-hand side of the landing page. Related content could include a list of categories and/or contact details for the media team. Another optional feature is “[Follow us](https://ons-design-system.netlify.app/styles/typography/#list-with-social-icons)” social media links to Instagram, Facebook, Twitter and LinkedIn.
+
+Add an image to your landing page. If you do not include an image, the system will use a placeholder. Placeholders do not look professional or support the written content to tell a story. The [articles component](/components/articles) has advice on image dimensions.
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/landing/index.njk" })
+}}
+
+### Article
+
+This is a text page that can include images and videos. It needs to have at least one of the categories: statement or press release.
+
+There is no limit on length. If you think an article seems too long, question the user need for such length with the content author.
+
+The title is important. It will stand out on the landing page and it links users to the article. It needs to:
+
+- be in plain English
+- use terms and phrases that reflect the language of users
+- be as short as possible
+- be frontloaded, with the most important information first
+
+Each article needs to have [Previous and Next skip links](/components/pagination#pager) at the end.
+
+You can add “[Share](/components/share-page)” social media links to Facebook, Twitter and LinkedIn.
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/article/index.njk" })
+}}
+
+### Tag
+
+Tags are not mandatory. Do not use the Tags block if there are no tags. A tag helps to identify an article, for example, “Census 2021”. Good tags help improve search performance. They need to be an important word or phrase that users will recognise. 
+
+You can use more than one tag for each article. Each tag is highlighted in yellow on the landing page summary. An “All posts [tag name]” landing page lists all the news posts that use that tag on the website. 
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/tag/index.njk" })
+}}
+
+### Category
```

Comment:
Assuming tags are only visible at the article level, I'd be inclined to bump the 'categories' section up above anything related to the article page.

Reply:
I would make this the first sub-heading in 'How to use this pattern' after describing how to use the landing page, and remove the example.  

So you'd have:
- Pattern title
- Lead in line
- [Main news page example]
- When to use this pattern
- How to use this pattern
- Categories
- [Category list page example]
- News articles
- [News article page example]
- Tags
- [Tag list page example]